src/composables/queries/usePoolQuery.ts

```diff
@@ -34,7 +36,7 @@ export default function usePoolQuery(
   /**
    * QUERY INPUTS
    */
-  const queryKey = QUERY_KEYS.Pools.Current(id);
+  const queryKey = QUERY_KEYS.Pools.Current(networkId, id);
```

Comment:
This is what I'm not sure about, but I suppose we can change it later when it becomes relevant. Given a pool ID we should always only load that pool. So I'm not sure what this networkId is going to represent, is it the user's networkId or is it the path defined networkId?

Reply:
You are right, I don't think we need `networId` here.